SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/Exceptions/SmtpException.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SwiftParcel.Services.Orders.Application.Exceptions
```

Comment:
Maybe, there is too small info here in the exception arguments

Reply:
I don't know if much can be done here but I will try to do something about this when i will add mailing for two other cases.